app/helpers/sufia_helper.rb

```diff
@@ -2,4 +2,19 @@ module SufiaHelper
   include ::BlacklightHelper
   include Sufia::BlacklightOverride
   include Sufia::SufiaHelperBehavior
+
+  def request_doi_link(asset, asset_type='Collection')
+    if asset[:identifier].empty? 
+      form_tag url_for(:controller => 'doi_requests', :action => 'create', :asset_id => asset.id, :asset_type => asset_type) do
```

Comment:
If we have the asset, do we need to pass in an asset type? Could we use `asset.class.to_s`?